warehouse/integrations/deltalake/deltalake.go

```diff
@@ -897,8 +899,22 @@ func (dl *Deltalake) AddColumns(tableName string, columnsInfo []warehouseutils.C
 	query += ");"
 
 	dl.Logger.Infof("DL: Adding columns for destinationID: %s, tableName: %s with query: %v", dl.Warehouse.Destination.ID, tableName, query)
-	err = dl.ExecuteSQL(query, "AddColumn")
-	return
+	executeResponse, err := dl.Client.Client.Execute(dl.Client.Context, &proto.ExecuteRequest{
+		Config:       dl.Client.CredConfig,
+		Identifier:   dl.Client.CredIdentifier,
+		SqlStatement: query,
+	})
+	if err != nil {
+		return fmt.Errorf("add columns: %w", err)
+	}
+
+	// Handle error in case of single column
+	if len(columnsInfo) == 1 {
```

Comment:
We would need to suppress the errors in case of a single column alter since databricks does support `ADD COLUMN IF NOT EXISTS`